cmd/backup_keys.go

```diff
@@ -133,7 +136,7 @@ func createTar(source string, target string) error {
 				return err
 			}
 
-			if !strings.HasSuffix(path, "charm_rsa") && !strings.HasSuffix(path, "charm_rsa.pub") {
+			if !strings.HasSuffix(path, "charm_ed25519") && !strings.HasSuffix(path, "charm_ed25519.pub") {
```

Comment:
@toby Is there any reason we'd want to also back up the old RSA keys? Assuming if people are using this version of `charm` mostly likely no, but figured worth flagging just incase.

Reply:
It could be cool too if someone has older keys that we migrate them to the ed25519 keys, then do a backup?

Reply:
Oh that’s so smart. Totally.